skops/card/_model_card.py

```diff
@@ -0,0 +1,60 @@
+from modelcards import CardData, ModelCard
+from sklearn.utils import estimator_html_repr
+
+
+def _extract_estimator_config(model):
+    """Extracts estimator configuration and renders them into a vertical table.
+
+    Parameters
+    ----------
+        model (estimator): scikit-learn pipeline or model.
+
+    Returns
+    -------
+    str:
+        Markdown table of hyperparameters.
+    """
+    hyperparameter_dict = model.get_params(deep=True)
+    table = "| Hyperparameters | Value |\n| :-- | :-- |\n"
+    for hyperparameter, value in hyperparameter_dict.items():
+        table += f"| {hyperparameter} | {value} |\n"
+    return table
+
+
+def create_model_card(
+    model, path, model_id=None, license=None, tags=None, metrics=None
+):
```

Comment:
I'm not sure about the API.

How is the user supposed to fill out sections which are text? I would expect an interface which would allow the user to insert all the sections that we're talking about with others in the modelcard team.

So maybe the API would be more like:

```python
card = skops.card.model_card_factory(...)
card.limitations = ...
card.author = ...
...
card.save(path/to/README.md, overwrite=True)
```

WDYT?

also pinging @nateraw here.

Reply:
isn't factory a bit overkill 😅 as for attributes, I agree.

Reply:
If we'll go with this I have questions:
1. Are we going with factory? If so, why? Can't we simply return default jinja template and let user inject values? 
2. How does output of model card factory differ from `default_template.md` (as a jinja template and not as a string)?

Reply:
To answer these questions, I think it makes sense for you to write down how you think the end user would interact with the object and functions we have in an example.

Whatever we choose, should be based on how the end user interacts with the library.

So maybe as the next step write a `plot_model_card.py` example with everything you thin the user can do, and we take it from there?

Reply:
I thought of the one I've already written in the `plot_hf_hub.py` but I'll write a more in depth one :)

Reply:
can't this `create_model_card` fn just take in the template kwargs if you want to override them? like limitations, authors, etc? Right now its not possible to override these after the class has been instantiated within `from_template`.  Also unsure if this card creation will be user-facing if its just going to be created automatically under the hood anyways.